README.md

```diff
@@ -74,9 +84,46 @@ replace (
 )
 ```
 
+#### Qitmeer is killed unexpectedly
+Qitmeer is killed probably due to 'Out of Memory' error, you may verify it by following command.
+```sh
+dmesg -T| grep -E -i -B100 'killed process'
+```
+and if you find similar output as follows, then that is the case
+```sh
+[Tue Mar  9 11:34:26 2021] Out of memory: Killed process 140587 (qitmeer) total-vm:1403144kB, 
+anon-rss:675828kB, file-rss:0kB, shmem-rss:0kB, UID:1001 pgtables:1532kB oom_score_adj:0
+```
+The minimum memory requirement is 1GB, and we strongly recommend upgrading the memory to 2GB.
+The other way MIGHT work is to upgrade your Golang to the latest version since the latest version incorporated memory 
+optimizations, and we received feedbacks from community that this is effective, especially for running Ubuntu 20 wth low
+memory.
+
+Please try following commands.  
+
+```sh
+sudo add-apt-repository ppa:longsleep/golang-backports
+sudo apt update
+sudo apt install golang-go
+```
+NOTE: not guarantee if working, the more recommended way is still upgrading memory.
+
+####  missing go.sum entry
```

Comment:
need not to do `go mod tidy` after #437 , refactor the doc to explain the issue && encourage user to update to the latest code when using go1.16 or above.

Reply:
edited